app/Resources/views/admin/blog/show.html.twig

```diff
@@ -5,6 +5,8 @@
 {% block main %}
     <h1>{{ post.title }}</h1>
 
+    {{ include('default/_flash_messages.html.twig') }}
```

Comment:
@stof, I think you're right. I've moved the displaying of flash messages to the main layout.